testsuite/scripts/gha_test_run_proto.json

```diff
@@ -153,6 +153,12 @@
             "test":"test_findText_notFound",
             "skip":true,
             "skipReason":"UnitTest sometimes fails when run in qpm"
-        }
+        },
+	{
```

Comment:
is the formatting correct? YOur code seems to be inlined differently.

Reply:
ah good catch @telephon , the new addition has tabs while the rest of the file uses spaces. @nuss could you change the new code to spaces please?

Reply:
hmm... I've seen that too in the diff. However, formatting looks correct in my forked repo: https://github.com/nuss/supercollider/blob/NodeProxy-seti/testsuite/scripts/gha_test_run_proto.json 
I've tried to keep my entry in line with other entries (indenting with spaces, not tabs). So, if formatting's not right I don't really know what's wrong.

Reply:
... sorry, I thought I had formatted my entry with tabs (in vim) but there were some tabs left.